static/templates/settings/data_exports_admin.hbs

```diff
@@ -1,5 +1,5 @@
 <div id="data-exports" class="settings-section" data-name="data-exports-admin">
-    <h3>{{t "Data exports" }}
+    <h3>{{t "Export data" }}
```

Comment:
We may want this to be "Export organization data" or something similar.

Reply:
Fixing.  I'm not sure what the best name is here, but I wanted something other than "Data exports" since I introduced it as the title of the list.  I tried not to go down the rabbit hole of fixing every wording issue that I encountered in the settings panels.  The wording right below that title is kind of awkward, but that's a problem for another day.